google/cloud/aiplatform/explain/__init__.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from google.cloud.aiplatform_v1beta1.types.explanation_metadata import (
+    ExplanationMetadata,
+)
+from google.cloud.aiplatform_v1beta1.types.explanation import ExplanationParameters
+
+from google.cloud.aiplatform_v1beta1.types.explanation import FeatureNoiseSigma
+
+# Classes used by ExplanationParameters
+from google.cloud.aiplatform_v1beta1.types.explanation import (
+    IntegratedGradientsAttribution,
+)
+from google.cloud.aiplatform_v1beta1.types.explanation import SampledShapleyAttribution
+from google.cloud.aiplatform_v1beta1.types.explanation import SmoothGradConfig
+from google.cloud.aiplatform_v1beta1.types.explanation import XraiAttribution
+
+# ExplanationMetadata subclasses
+InputMetadata = ExplanationMetadata.InputMetadata
+OutputMetadata = ExplanationMetadata.OutputMetadata
+
+# InputMetadata subclasses
+Encoding = ExplanationMetadata.InputMetadata.Encoding
+FeatureValueDomain = ExplanationMetadata.InputMetadata.FeatureValueDomain
+Visualization = ExplanationMetadata.InputMetadata.Visualization
+
```

Comment:
a bit orthogonal question: why don't we have Modality class?

Reply:
It appears that `modality` is a string field in `InputMetadata` [according to the protos](http://google3/google/cloud/aiplatform/master/explanation_metadata.proto?l=289-294&rcl=358240862). Should it be an enum?

Reply:
I was looking at [this proto](https://source.corp.google.com/piper///depot/google3/cloud/ai/platform/db/proto/explanation.proto;l=250). Do you know why it is an Enum in the DB one and not in the other by any chance? Anyhow, lgtm.

Reply:
Interesting, I raised this with jianan@ and he explained they lean towards string over enums in the API protos to avoid having to make frequent changes to those files while the DB protos are more strongly typed. If it improves usability, we could manually add the enum in the SDK but it would have to be kept in sync.